src/CONST.ts

```diff
@@ -1008,6 +1008,12 @@ const CONST = {
         VIDEO: 'video',
     },
 
+    FILE_FORMAT: {
```

Comment:
```suggestion
    IMAGE_MIME_TYPES: {
```

or `IMAGE_FILE_FORMAT`